exotica_python/src/Exotica.cpp

```diff
@@ -596,11 +596,40 @@ PYBIND11_MODULE(_pyexotica, module)
     kdlFrame.def("getTranslationAndAngleAxis", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::ANGLE_AXIS); });
     kdlFrame.def("getTranslationAndQuaternion", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::QUATERNION); });
     kdlFrame.def("getFrame", [](KDL::Frame* me) { return getFrame(*me); });
-    kdlFrame.def("inverse", (KDL::Frame (KDL::Frame::*)() const) & KDL::Frame::Inverse);
+    kdlFrame.def("inverse", (KDL::Frame(KDL::Frame::*)() const) & KDL::Frame::Inverse);
     kdlFrame.def("__mul__", [](const KDL::Frame& A, const KDL::Frame& B) { return A * B; }, py::is_operator());
+    kdlFrame.def_readwrite("p", &KDL::Frame::p);
```

Comment:
`p` is a public member of `KDL::Frame`, so you can read/write them in C++ without setter/getters and KDL actually does not provide these for `Frame`.
In python, an error will be thrown if you try to write anything else than a `KDLVector`.

Reply:
Okay!

Reply:
What happens if you write a numpy of size 4 (or not 3) to it? Will it throw? Or corrupt?

Reply:
Yea, it will recognise the wrong type:
```
Traceback (most recent call last):
  File "[...]/exotica/examples/exotica_examples/scripts/example_attach.py", line 32, in <module>
    T.p = np.array([0,0,-0.5, 23, 56])
TypeError: (): incompatible function arguments. The following argument types are supported:
    1. (self: _pyexotica.KDLFrame, arg0: KDL::Vector) -> None

Invoked with: KDL::Frame ([0.000000 0.000000 0.000000] [0.000000 0.000000 0.000000 1.000000]), array([ 0. ,  0. , -0.5, 23. , 56. ])
```
Where `KDL::Frame` is `self` and `array` is the wrong type that was used in place of the correct `KDL::Vector`.